cat-launcher/src/pages/PlayPage/index.tsx

```diff
@@ -13,12 +13,23 @@ import {
   sortableKeyboardCoordinates,
   verticalListSortingStrategy,
 } from "@dnd-kit/sortable";
+import { usePostHog } from "@posthog/react";
+import { useEffect } from "react";
 
 import { useGameVariants } from "@/hooks/useGameVariants";
+import { trackPageView } from "@/lib/analytics";
 import { toastCL } from "@/lib/utils";
 import GameVariantCard from "./GameVariantCard";
 
 function PlayPage() {
+  const posthog = usePostHog();
```

Comment:
Combine the usePostHog and the related useEffect into a single hook call: useTrackPageView.